crates/provider/src/shared/types.rs

```diff
@@ -11,7 +11,9 @@ pub type ExecutionResult = Result<String, (ExitStatus, String)>;
 
 #[derive(Debug, Clone, PartialEq)]
 pub struct ProviderCapabilities {
+    /// Ensure that we have an image for each node (k8s/podman/docker)
```

Comment:
We use this flag to decide other things, so maybe we need to rename it.

Reply:
Yes, I saw above, maybe we could split this in multiple flags to avoid confusion and keep one flag for one thing